src/components/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./Button.module.scss";
+
+const Button = ({ onClick, children, disabled }) => {
+  return (
+    <button
+      // if the button has the disabled class on it, add the disabled styling to it
+      className={`${styles.btn} ${disabled ? styles.disabled : ""}`}
+      onClick={onClick}
+      disabled={disabled}
+    >
```

Comment:
instead of passing onClick and disabled you can do 
```jsx
const Button = ({ children, ...rest }) => {
  return (
    <button
     {...rest}
      className={styles.btn}
    >
    {children}
    </button>
```
and use :disabled psuedo-class in your scss